catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/source/impl/SourceDescriptorImpl.java

```diff
@@ -95,7 +95,7 @@ public void setAvailable(boolean isAvailable) {
 
     @Override
     public Date getLastAvailabilityDate() {
-        return lastAvailableDate;
+        return (Date)lastAvailableDate.clone();
```

Comment:
Yes, thanks for the reminder. I realized later on that I should be using the millis but forgot to go back and remove the clone() methods.